features/10_form-labels.md

```diff
@@ -1,47 +0,0 @@
-# Correctly use form labels
-
-Form fields that allow input (`select`, and `textarea` elements, and all `input` element types other than image, submit, reset, button, or hidden) **must** have an associated label, either in the form of a `<label>` element or, for simple forms when no visible label is required, a `title` attribute.
-
-## Rationale
-
-Form labels are important for all users in order to understand what the form field is however they are essential for speech output users who cannot easily infer what the form element is by looking at the surrounding content.
-
-While there are WAI-ARIA attributes that allow for labelling of forms it is not supported in all versions of assistive technologies that BBC users could reasonably expect to be able to use.
-
-## Techniques
-
-Pass:
-
-	<label for="search">Search the BBC</label>
-	<input type="text" id="search" name="q" />
-
-	<label for="search">
-		Search the BBC
-		<input type="text" name="q" />
-	</label>
-
-	<input type="text" name="q" title="Search the BBC" />
-
-Fail:
-
-	<input type="text" name="name" title="Name" />
-	<input type="text" name="email" title="Email" />
```

Comment:
@IanPouncey I'm puzzled why these two are in the fail category - according to the rule as I understand it they look OK - they have a `title` attribute. Is this something to do with the subjective phrase above "simple forms when no visible label is required"? If so, could we say "forms with only one field" rather than "simple forms"?